iphone/Classes/MediaModule.m

```diff
@@ -1863,7 +1863,8 @@ - (void)imagePickerController:(UIImagePickerController *)picker_ didFinishPickin
             }
             
             if (resultImage == nil) {
-                resultImage = (editedImage != nil) ? editedImage : originalImage;
+                UIImage *tempImage = (editedImage != nil) ? editedImage : originalImage;
+                resultImage = [TiUtils adjustRoation:tempImage];
```

Comment:
I originally had a typo in the method name changed the method but forgot to change its call